playbooks/roles/ocp-cost-management/tasks/cmo-setup.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Setup and Test Cost Management Metrics Operator
+  hosts: localhost

```

Comment:
"host" parameter is not required here, please remove it

Reply:
done